packages/bitcore-node/src/routes/api/fee.ts

```diff
@@ -30,6 +34,14 @@ router.get('/:target', CacheMiddleware(CacheTimes.Second), async (req: Request,
   } catch (err) {
     return res.status(500).send('Error getting fee from RPC');
   }
+}
+
+router.get('/:target', CacheMiddleware(CacheTimes.Second), async (req: Request, res: Response) => {
+  getFee(req, res);
+});
+
+router.get('/:target/:txType', CacheMiddleware(CacheTimes.Second), async (req: Request, res: Response) => {
```

Comment:
this is minor, but i think it would be a little cleaner and scalable to add a `txType` query param to the existing route rather making txType a path param. (side note: Express does have syntax for optional path params, but i would recommend not using those except in rare cases.)